torch/onnx/symbolic_helper.py

```diff
@@ -173,7 +173,12 @@ def decorator(fn):
         @wraps(fn)
         def wrapper(g, *args, **kwargs):
             # some args may be optional, so the length may be smaller
-            assert len(arg_descriptors) >= len(args)
+            # if fn.__name__ == 'cat':
+            #     import pdb; pdb.set_trace()
+            assert len(arg_descriptors) >= len(args),\
+                f"A mismatch between the number of arguments ({len(args)}) and"\
+                f" their descriptors ({len(arg_descriptors)}) was found at function '{fn.__name__}'"
```

Comment:
curious would it be possible if fn does not have attribute `__name__`?

Reply:
On top of that, I feel we could provide actionable suggestions in this message. e.g. submit bug for official symbolic implementation, or fix custom symbolic implementation. Perhaps even differentiate if the symbolic is custom or not, but that could be a bit too complicated.

Reply:
There will always be a `__name__`. In fact, the main goal of `@wraps(fn)` is to copy the `fn.__name__` and `fn.__docstring__` into `wrapper`'s

Reply:
> On top of that, I feel we could provide actionable suggestions in this message. e.g. submit bug for official symbolic implementation, or fix custom symbolic implementation. Perhaps even differentiate if the symbolic is custom or not, but that could be a bit too complicated.

One of the issues in being actionable is that the issue might be happening on custom ops and contrib ops. This is exactly what triggered this PR. ONNX Runtime custom symbolics differed from PyTorch's after migrating from 1.9.0 to 1.10

I will try rewrite the error so it explicitly says the symbolic is broken.